binding/web/src/leopard.ts

```diff
@@ -199,15 +146,34 @@ export class Leopard {
    * it can create an instance.
    *
    * @param accessKey AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)
-   * @param modelPath Path to the model saved in indexedDB.
-   * @param options Optional configuration arguments.
+   * @param model Leopard model options.
+   * @param model.base64 The model in base64 string to initialize Leopard.
+   * @param model.publicPath The model path relative to the public directory.
+   * @param model.customWritePath Custom path to save the model in storage.
+   * Set to a different name to use multiple models across `leopard` instances.
+   * @param model.forceWrite Flag to overwrite the model in storage even if it exists.
+   * @param model.version Leopard model version. Set to a higher number to update the model file.
```

Comment:
Gotta be careful calling this `Leopard model version`. Folks may get it confused with the version of Leopard. Maybe just say `Version of the model file. Increment to update the model file in storage.`